src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -68,6 +73,10 @@ static App()
 		/// </summary>
 		public App()
 		{
+#if !HAS_UNO_WINUI
+			Assert.IsNotNull(Windows.UI.Xaml.Window.Current);
```

Comment:
Accessing `Current` will force-initialize the window if it's not already initialized.

So, I think if there is a bug that causes window to not be initialized at the right timing, this assert might mask such bug.